README.md

```diff
@@ -253,13 +253,13 @@ Please ensure you have the required [CUDA dependency](#cuda-dependency).
 
 The Common Voice corpus consists of voice samples that were donated through Mozilla's [Common Voice](https://voice.mozilla.org/) Initiative.
 
-We provide an importer (`bin/import_cv.py`) which automates downloading and preparing the Common Voice corpus as such:
+We provide an importer (`bin/import_cv2.py`) which automates downloading and preparing the Common Voice (v2.0) corpus as such:
 
 ```bash
-bin/import_cv.py path/to/target/directory
+bin/import_cv2.py path/to/target/directory
 ```
 
-If you already downloaded Common Voice from [here](https://voice.mozilla.org/data), simply run `bin/import_cv.py` on the directory where the corpus is located. The importer will detect that you've already downloaded the data and immediately proceed to unpackaging and importing. If you haven't downloaded the data already, `bin/import_cv.py` will download it for you and save to the path you've specified.
+If you already downloaded Common Voice from [here](https://voice.mozilla.org/data), simply run `bin/import_cv2.py` on the directory where the corpus is located. The importer will detect that you've already downloaded the data and immediately proceed to unpackaging and importing. If you haven't downloaded the data already, `bin/import_cv2.py` will download it for you and save to the path you've specified.
```

Comment:
> ...simply run `bin/import_cv2.py` on the directory where the corpus is located...

This is not how `import_cv2.py` works. `import_cv2.py` requires two arguments, see [here](https://github.com/mozilla/DeepSpeech/blob/master/bin/import_cv2.py#L139).

Reply:
12d31c11bde9b214b703e0cd44cfb9e3ce0b14a8